### PR TITLE
Add manual setting of fatigue_limit

### PR DESCRIPTION
**Pull request by old2rng**
Add enhancement #73

### CODE REVIEW

File: src/pylife/materialdata/woehler/fatigue_data.py [hunk 1]
```diff
@@ -142,6 +142,31 @@ def conservative_fatigue_limit(self):
 
         return self
 
+    def set_fatigue_limit(self, fatigue_limit=None):
```
Comment (johannes-mueller) [inline src/pylife/materialdata/woehler/fatigue_data.py:145]: Why the `None` default? I would assume that whoever calls the method wants to set it. Allowing `None` makes the whole function way more complicated. Do we need it?

File: tests/materialdata/woehler/test_analyzer.py [hunk 2]
```diff
@@ -82,17 +82,32 @@ def test_fatigue_data_finite_infinite_zone(data, finite_zone_expected, infinite_
...
+    (data,
+     finite_expected_set_finite_limit_max,
+     infinite_expected_set_finite_limit_max,
+     fat_limit_max),
+    (data,
+     finite_expected_set_finite_limit_min,
+     infinite_expected_set_finite_limit_min,
+     fat_limit_min)
+])
+def test_fatigue_data_finite_infinite_zone_manuel_setting(data, finite_zone_expected, infinite_zone_expected, fat_limit):
```
Comment (johannes-mueller) [inline tests/materialdata/woehler/test_analyzer.py:105]: Typo:

```suggestion
def test_fatigue_data_finite_infinite_zone_manual_setting(data, finite_zone_expected, infinite_zone_expected, fat_limit):
```

**Review by johannes-mueller — CHANGES_REQUESTED** [review]
Thanks @old2rng 

Two remarks in the comments.

File: src/pylife/materialdata/woehler/fatigue_data.py (hunk 1 above)
Comment (old2rng) [reply src/pylife/materialdata/woehler/fatigue_data.py:145]: We don't "really" need it. I assumed people might forget to add a value. I guess it is better to get rid of it.

**Review by johannes-mueller — APPROVED** [review]
Thanks, merging

**Pull request merged**